cmsplugin_zinnia/admin.py

```diff
@@ -17,8 +16,10 @@ class EntryPlaceholderAdmin(PlaceholderAdminMixin, EntryAdmin):
     EntryPlaceholder Admin
     """
     fieldsets = (
-        (_('Content'), {'fields': (('title', 'status'), 'image')}),) + \
```

Comment:
image duplicates are not allowed and current forms for the image only display the latter field